test/test_comment.py

```diff
@@ -66,8 +68,14 @@ def _test_ops(
                         coll.insert_one({})
                         maybe_cursor = db.validate_collection(*args, **kwargs)
                     else:
-                        coll.create_index("a")
-                        maybe_cursor = h(*args, **kwargs)
+                        if not _IS_SYNC and isinstance(coll, Empty):
+                            coll.create_index("a")
```

Comment:
How can we call create_index when coll is `Empty`?

Reply:
Thats a great question...HAHA code is removed now.